nixos/tests/ceph.nix

```diff
@@ -0,0 +1,140 @@
+import ./make-test.nix ({pkgs, ...}: rec {
+  name = "All-in-one-basic-ceph-cluster";
+  meta = with pkgs.stdenv.lib.maintainers; {
+    maintainers = [ lejonet ];
```

Comment:
typo in the name?

Reply:
Whatcha mean?

Reply:
That handle does not exists in `lib/maintainers.nix`

Reply:
Fixed: https://github.com/NixOS/nixpkgs/pull/35299/commits/29e78ce9a492d8709eb40f54f3aac1c0c97e3fe0